lib/elixir/pages/Writing Documentation.md

```diff
@@ -53,33 +53,41 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
 
-  * Markdown uses backticks (`` ` ``) to quote code. Elixir builds on top of that to automatically generate links when module or function names are referenced. For this reason, always use full module names. If you have a module called `MyApp.Hello`, always reference it as `` `MyApp.Hello` `` and never as `` `Hello` ``. Function names must be referenced by name and arity if they are local, as in `` `world/1` ``, or by module, name and arity if pointing to an external module: `` `MyApp.Hello.world/1` ``. Referencing a `@callback` can be done by prepending `c:`, as in `` `c:world/1` ``; and a `@type` by using `t:`, as in `t:values/0`.
+  * Reference modules by their full name.
 
-  * When the documentation has multiple sections, always start the section heading by using `##`. The first heading is reserved for modules and function names.
+  Markdown uses backticks (`` ` ``) to quote code. Elixir builds on top of that to automatically generate links when module or function names are referenced. For this reason, always use full module names. If you have a module called `MyApp.Hello`, always reference it as `` `MyApp.Hello` `` and never as `` `Hello` ``.
 
-  * When documenting a function with multiple clauses, the documentation must be placed before the first clause. Documentation is always per function and arity and not per clause.
+  * Reference functions by name and arity if they are local, as in `` `world/1` ``, or by module, name and arity if pointing to an external module: `` `MyApp.Hello.world/1` ``.
+
+  * Reference a `@callback` by prepending `c:`, as in `` `c:world/1` ``.
+
+  * Reference a `@type` by prepending `t:`, as in `` `t:values/0` ``.
+
+  * Start new sections with second level markdown headings `##`. The first heading is reserved for module and function names.
+
+  * Place documentation before the first clause of multi-clause functions. Documentation is always per function and arity and not per clause.
 
 ## Doctests
 
-We recommend developers to include examples in their documentation, often under its own `## Examples` heading. To ensure examples do not get out of date, Elixir's test framework (ExUnit) provides a feature called doctests that allows developers to test the examples in their documentation. Doctests work by parsing out code samples starting with `iex>` from the documentation. You can read more about it at `ExUnit.DocTest`.
+We recommend developers include examples in their documentation, often under their own `## Examples` heading. To ensure examples do not get out of date, Elixir's test framework (ExUnit) provides a feature called doctests that allows developers to test the examples in their documentation. Doctests work by parsing out code samples starting with `iex>` from the documentation. You can read more about it at `ExUnit.DocTest`.
```

Comment:
"recommend developers include" is correct over "recomment developers to include", but maybe "We recommend *that* developers include" reads even nicer?